Update Strimzi version

adity1raut opened the pull request:
Solve Issue #6918

Update the version of StrimziVersion in the file tests/helper/helper.go:

     - StrimziVersion = "0.35.0"  
     + StrimziVersion = "0.47.0"
Also, update other relevant files such as go.mod and go.sum to reflect this version change

adity1raut commented:
Hi @rickbrouwer , could you please take a moment to review this PR at your convenience? I would greatly appreciate your feedback.

rickbrouwer commented:
> Hi @rickbrouwer , could you please take a moment to review this PR at your convenience? I would greatly appreciate your feedback.

Hi @adity1raut ,

As soon as you create a merge request, you'll see a merge request template. This template provides information so you can provide a clear description of the issue you're solving. It would be helpful if you could leave it intact and use it, as it helps us with the review. It looks like you've completely removed this.

I also see that you've made some changes to `go.mod` and `go.sum`. You've also added something to `.gitignore` that isn't necessary for this change.

The comment in the previous review hasn't been answered yet: "Why is the addition of get_helm necessary?"

adity1raut commented:
> > Hi @rickbrouwer , could you please take a moment to review this PR at your convenience? I would greatly appreciate your feedback.
> 
> Hi @adity1raut ,
> 
> As soon as you create a merge request, you'll see a merge request template. This template provides information so you can provide a clear description of the issue you're solving. It would be helpful if you could leave it intact and use it, as it helps us with the review. It looks like you've completely removed this.
> 
> I also see that you've made some changes to `go.mod` and `go.sum`. You've also added something to `.gitignore` that isn't necessary for this change.
> 
> The comment in the previous review hasn't been answered yet: "Why is the addition of get_helm necessary?"

The get_helm script is not strictly necessary. I added it while updating the code because my GitHub Actions workflow was failing , will Delete this file

adity1raut commented:
> > Hi @rickbrouwer , could you please take a moment to review this PR at your convenience? I would greatly appreciate your feedback.
> 
> Hi @adity1raut ,
> 
> As soon as you create a merge request, you'll see a merge request template. This template provides information so you can provide a clear description of the issue you're solving. It would be helpful if you could leave it intact and use it, as it helps us with the review. It looks like you've completely removed this.
> 
> I also see that you've made some changes to `go.mod` and `go.sum`. You've also added something to `.gitignore` that isn't necessary for this change.
> 
> The comment in the previous review hasn't been answered yet: "Why is the addition of get_helm necessary?"

For the .gitignore file, I used a Python environment to resolve issues related to the pre-commit run --all-files command. I want to update the version, so I’m modifying this file

adity1raut commented:
> > Hi @rickbrouwer , could you please take a moment to review this PR at your convenience? I would greatly appreciate your feedback.
> 
> Hi @adity1raut ,
> 
> As soon as you create a merge request, you'll see a merge request template. This template provides information so you can provide a clear description of the issue you're solving. It would be helpful if you could leave it intact and use it, as it helps us with the review. It looks like you've completely removed this.
> 
> I also see that you've made some changes to `go.mod` and `go.sum`. You've also added something to `.gitignore` that isn't necessary for this change.
> 
> The comment in the previous review hasn't been answered yet: "Why is the addition of get_helm necessary?"

I used the latest Go version 1.24.1. After changing StrimziVersion = "0.47.0", I ran the go mod tidy command, which automatically updated the dependencies

adity1raut commented:
> environment

I will changed

adity1raut commented:
> `et_helm`

Sir, get-helm is not necessary. If you don't want it, I can delete it

adity1raut commented:
> > > `et_helm`
> > 
> > 
> > Sir, get-helm is not necessary. If you don't want it, I can delete it
> 
> yes, you can delete it

Sir, if there are any other changes you'd like me to make, please let me know

adity1raut commented:
> > > > > `et_helm`
> > > > 
> > > > 
> > > > Sir, get-helm is not necessary. If you don't want it, I can delete it
> > > 
> > > 
> > > yes, you can delete it
> > 
> > Sir, if there are any other changes you'd like me to make, please let me know
> 
> Please also undo the dependency updates (go.mod en go.sum)

This dependency is necessary for this version

rickbrouwer commented:
> > > > > > `et_helm`
> > > > > 
> > > > > 
> > > > > Sir, get-helm is not necessary. If you don't want it, I can delete it
> > > > 
> > > > 
> > > > yes, you can delete it
> > > 
> > > 
> > > Sir, if there are any other changes you'd like me to make, please let me know
> > 
> > 
> > Please also undo the dependency updates (go.mod en go.sum)
> 
> This dependency is necessary for this version

Can you explain me for better understanding.
If you run a `go mod tidy` against the current main branch without your update, will these dependency updates be included as well?

adity1raut commented:
@rickbrouwer  Could you please check this PR and share your review?

rickbrouwer reviewed (approved):
yes. lgtm!

SpiritZhou commented:
/run-e2e kafka

JorTurFer commented:
/run-e2e kafka
